svm-transaction/src/svm_message/sanitized_message.rs

```diff
@@ -0,0 +1,66 @@
+use {
+    crate::{Instruction, MessageAddressTableLookup, SVMMessage},
+    solana_sdk::{
+        hash::Hash,
+        message::{AccountKeys, SanitizedMessage},
+        pubkey::Pubkey,
+    },
+};
+
+// Implement for the "reference" `SanitizedMessage` type.
+impl SVMMessage for SanitizedMessage {
+    fn num_signatures(&self) -> u64 {
+        SanitizedMessage::num_signatures(self)
```

Comment:
Not that this is wrong, but why did you use in some places `self.function()`, but in others `SanitizedMessage::function(self)`?

Reply:
I'll update to just be consistent between all of them.

Some of these names are fns on the actual type it's non-trait `impl` block, but share the exact same name with fns in the current trait we're implementing.
Was using the `SanitizedMessage::XXX` syntax to specify to rust "use the non-trait fn, instead of calling this trait function recursively".

Others did not share names, so I just called them with the `self.XXX` syntax.

Think you're right that it'd be better if the syntax was just consistent across all the impls here.

Reply:
done: f9a70267f0